rs-backend/src/main.rs

```diff
@@ -12,6 +12,7 @@ extern crate serde_json;
 
 pub const SERVER_ADDRESS: &'static str = "0.0.0.0:2346";
 
+mod handler;
```

Comment:
this mod should have a more descriptive name - handler is a bit generic. hook_handler? or notification_handler or something.